packages/ui-extensions/docs/surfaces/point-of-sale/reference/examples/targets/pos-register-details-block-render.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+
+import {
+  POSBlock,
+  Text,
+  POSBlockRow,
+  reactExtension,
+  useApi,
+} from '@shopify/ui-extensions-react/point-of-sale';
+
+const Block = () => {
+  const api = useApi<'pos.register-details.block.render'>();
+
+  return (
+    <POSBlock
+      action={{
+        title: 'Open Cash Management App',
+        onPress: api.action.presentModal,
+      }}
+    >
+      <POSBlockRow>
+        <Text>This is a block extension</Text>
```

Comment:
ive never looked at extensions code before. but just confirming there isnt any translation file or anything for this stuff? i know its just an example file